andela/knights-bn-backend
# #171947441  improve test coverage on returnTrip.js

#### What does this PR do?

- Improve code coverage of the overall project and fix some bugs.

#### Description of Task to be completed?

- Test return trip middleware and its dependencies.
- Fix and make sendEmail.js re-usable
- Fix view all accommodation
- Fix socialLogin missing strategies

#### How should this be manually tested?

- run `git clone https://github.com/andela/knights-bn-backend.git` to clone the repo.

- run `cd knights-bn-backend` in your terminal to enter into the repository.

- create two Postgres database for development and testing respectively.

- run `git checkout -t bg-improve-test-validateReturnTrip-171947441` to switch on this branch.

- run `npm run createTables` to create tables for you

- run `npm install` to install dependencies.

- create a .env file then add environment variable referring to ` .env.example`,  check our slack channel `code-knights-12` for .env.


- run `npm run test` to test.

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [link](https://www.pivotaltracker.com/story/show/171947441)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A
